service/src/test/resources/ehr/mapper/observation/pertinent/example-observation-resource-with-pert-info-1.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
Something to consider for the JSON fixtures for these "sub-mapper". You can probably get away with using minimal resources here e.g. removing all fields except for referenceRange or value[x] as the case may be. It would make these files much smaller.